nautilus_trader/common/actor.pyx

```diff
@@ -1397,15 +1398,18 @@ cdef class Actor(Component):
         await_partial : bool, default False
             If the bar aggregator should await the arrival of a historical partial bar prior
             to actively aggregating new bars.
+        notify_actor : bool, default True
```

Comment:
It seems asymmetric to add this just for bars? When would an actor call `subscribe_bars` but not want to actually receive them on the bus?

Could this be custom functionality we're asking regular actors to accommodate?

Reply:
Removed as discussed